### PR TITLE
[WIP] Add Brussletor benchmarks for nonlinear solvers

### PR DESCRIPTION
**Pull request by ErikQQY**
Add Brusselator benchmarks for all the nonlinear solvers

### CODE REVIEW

File: benchmarks/NonlinearProblem/Bruss_wpd.jmd [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+```julia
+using NonlinearSolve, NonlinearSolveMINPACK, SciMLNLSolve, SimpleNonlinearSolve, Sundials
+using LinearAlgebra, SparseArrays, DiffEqDevTools, Plots
+RUS = RadiusUpdateSchemes;
+```
+
+Define the Brussletor problem.
+
+```julia
+const N = 32
```
Comment (avik-pal) [inline benchmarks/NonlinearProblem/Bruss_wpd.jmd:19]: Lets add an axis for increasing the discretization.

Comment (ErikQQY) [reply benchmarks/NonlinearProblem/Bruss_wpd.jmd:19]: Increasing N to increase the discretization? Wouldn't that increases the benchmarking time?

Comment (ChrisRackauckas) [reply benchmarks/NonlinearProblem/Bruss_wpd.jmd:19]: yes, but we got a big machine

File: benchmarks/NonlinearProblem/Bruss_wpd.jmd [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+```julia
+using NonlinearSolve, NonlinearSolveMINPACK, SciMLNLSolve, SimpleNonlinearSolve, Sundials
+using LinearAlgebra, SparseArrays, DiffEqDevTools, Plots
+RUS = RadiusUpdateSchemes;
+```
+
+Define the Brussletor problem.
+
+```julia
+const N = 64
```
[the remark this answers is not in the document]
Comment (ErikQQY) [reply benchmarks/NonlinearProblem/Bruss_wpd.jmd:19]: So we need to increase both N and the order of tolerance to make sure the algorithms scale?

**Comment by ChrisRackauckas** [comment]
We can do this in two steps. Let's first get one in with a fixed N. Then let's do one where we loop over N and show the difference of the methods.

**Pull request closed without merging**